.chronus/changes/tcgc-fixInternalParameterizedNextLink-2025-2-19-12-18-13.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@azure-tools/typespec-azure-core"
+---
+
+Expose `getParameterizedNextLinkParams`
```

Comment:
the decorator was merged after the release right? if so I would change the  kind to `internal`